src/uct/sm/cma/cma_iface.c

```diff
@@ -77,13 +77,10 @@ static UCS_CLASS_INIT_FUNC(uct_cma_iface_t, uct_md_h md, uct_worker_h worker,
                            const uct_iface_params_t *params,
                            const uct_iface_config_t *tl_config)
 {
-    size_t sys_max_iov = 0;
-
     UCS_CLASS_CALL_SUPER_INIT(uct_base_iface_t, &uct_cma_iface_ops, md, worker,
                               tl_config UCS_STATS_ARG(NULL));
 
-    sys_max_iov = ucs_max(sysconf(_SC_IOV_MAX), 1);
-    self->super.config.max_iov = ucs_min(UCT_SM_MAX_IOV, sys_max_iov);
+    self->super.config.max_iov = ucs_min(UCT_SM_MAX_IOV, ucs_get_max_iov());
```

Comment:
we can add this as a common function to sm_iface.h:
uct_sm_get_max_iov() { return ucs_min(UCT_SM_MAX_IOV, ucs_get_max_iov()); }